rca/static_src/sass/abstracts/_variables.scss

```diff
@@ -117,6 +117,9 @@ $gutter: $grid;
 // Wrappers
 $site-width: 1440px;
 
+// Offsets
+$image-offset-right: -($gutter * 3);
```

Comment:
I decided to make a variable for this since there is quite a lot of the UI where we want this offset to be the same.